Sprintfinity3902/Entities/Items/SwordHitboxItem.cs

```diff
@@ -62,7 +62,7 @@ public override void Update(GameTime gameTime)
 
         public void MoveItem()
         {
-            if (itemUseCount <= 5) //this time amount needs to be tweaked.
+            if (itemUseCount <= 30) //this time amount needs to be tweaked.
```

Comment:
At some point this Magic Number might need to be taken out.

Reply:
Thanks for catching this, just gave it a constant.